[2024-08-08 08:06:00,267] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dota_dag.transform_heroes 2024-08-08T08:00:56.986609+00:00 [queued]>
[2024-08-08 08:06:00,284] {taskinstance.py:877} INFO - Dependencies all met for <TaskInstance: dota_dag.transform_heroes 2024-08-08T08:00:56.986609+00:00 [queued]>
[2024-08-08 08:06:00,284] {taskinstance.py:1068} INFO - 
--------------------------------------------------------------------------------
[2024-08-08 08:06:00,285] {taskinstance.py:1069} INFO - Starting attempt 1 of 2
[2024-08-08 08:06:00,285] {taskinstance.py:1070} INFO - 
--------------------------------------------------------------------------------
[2024-08-08 08:06:00,304] {taskinstance.py:1089} INFO - Executing <Task(PythonOperator): transform_heroes> on 2024-08-08T08:00:56.986609+00:00
[2024-08-08 08:06:00,309] {standard_task_runner.py:52} INFO - Started process 69 to run task
[2024-08-08 08:06:00,313] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'dota_dag', 'transform_heroes', '2024-08-08T08:00:56.986609+00:00', '--job-id', '13', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/dota.py', '--cfg-path', '/tmp/tmp0vfjoztb', '--error-file', '/tmp/tmpc8gp0i64']
[2024-08-08 08:06:00,314] {standard_task_runner.py:77} INFO - Job 13: Subtask transform_heroes
[2024-08-08 08:06:00,361] {logging_mixin.py:104} INFO - Running <TaskInstance: dota_dag.transform_heroes 2024-08-08T08:00:56.986609+00:00 [running]> on host ec67c4f7e31a
[2024-08-08 08:06:00,402] {taskinstance.py:1283} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=dota_dag
AIRFLOW_CTX_TASK_ID=transform_heroes
AIRFLOW_CTX_EXECUTION_DATE=2024-08-08T08:00:56.986609+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-08-08T08:00:56.986609+00:00
[2024-08-08 08:06:00,454] {python.py:118} INFO - Done. Returned value was: None
[2024-08-08 08:06:00,470] {taskinstance.py:1192} INFO - Marking task as SUCCESS. dag_id=dota_dag, task_id=transform_heroes, execution_date=20240808T080056, start_date=20240808T080600, end_date=20240808T080600
[2024-08-08 08:06:00,520] {taskinstance.py:1246} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-08 08:06:00,565] {local_task_job.py:146} INFO - Task exited with return code 0
